### PR TITLE
Remove references to an old function.

### LINKED ISSUE #4643: Remove references to compute_support_points
I believe that this function has been replaced by `compute_mapping_support_points`, but we still mention it in `mapping_c1.h` and `mapping_q_generic.h`.

### PR DESCRIPTION
**Pull request by drwells**
A little PR that just updates some of the documentation.

Fixes #4643.

### CODE REVIEW

File: include/deal.II/fe/mapping_q_generic.h [hunk 1]
```diff
@@ -692,16 +692,15 @@ class MappingQGeneric : public Mapping<dim,spacedim>
...
-   * Needed by the @p compute_support_points() function. For <tt>dim=1</tt>
-   * this function is empty. The function uses the underlying manifold object
-   * of the line (or, if none is set, of the cell) for the location of the
-   * requested points.
+   * This method uses the underlying manifold object of the line (or, if none
+   * is set, of the cell) for the location of the requested points. This
+   * method is usually called by compute_mapping_support_points() function.
    *
-   * This function is made virtual in order to allow derived classes to choose
+   * This method is made virtual in order to allow derived classes to choose
```
Comment (bangerth) [inline include/deal.II/fe/mapping_q_generic.h]: Technically (from the perspective of programming languages), this is correct. But C++ nevertheless talks about "member functions", so I would stick with the original naming.

File: include/deal.II/fe/mapping_q_generic.h [hunk 2]
```diff
@@ -711,16 +710,16 @@ class MappingQGeneric : public Mapping<dim,spacedim>
...
-   * Needed by the @p compute_support_points() function. For <tt>dim=1</tt>
-   * and <tt>dim=2</tt> this function is empty. The function uses the
-   * underlying manifold object of the quad (or, if none is set, of the cell)
-   * for the location of the requested points.
+   * This method uses the underlying manifold object of the quad (or, if none
+   * is set, of the cell) for the location of the requested points. This
+   * method is usually called by compute_mapping_support_points().
    *
-   * This function is made virtual in order to allow derived classes to choose
+   * This method is made virtual in order to allow derived classes to choose
```
Comment (bangerth) [inline include/deal.II/fe/mapping_q_generic.h]: same here

**Comment by drwells** [comment]
Good point. I used 'function' instead of 'method' in the new text.

**Pull request merged**